it.mijorus.gearlever.json

```diff
@@ -0,0 +1,75 @@
+{
+    "app-id" : "it.mijorus.gearlever",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "gearlever",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--device=dri",
+
+        "--filesystem=xdg-data/applications:rw",
+        "--filesystem=xdg-data/icons:ro",
+        "--filesystem=home:rw",
+
+        "--talk-name=org.freedesktop.FileManager1",
+        "--talk-name=org.freedesktop.Flatpak",
```

Comment:
this dbus is not allowed by default.

Reply:
Which one? The ".Flatpak"? Some apps on Flathub have it

Reply:
and they either have an exception of haven't been rebuilt in a long time.

Reply:
How can I launch external apps without using that portal?

Reply:
you need to apply for an exception. see https://github.com/flathub/flatpak-builder-lint/pull/81/files for an example

Reply:
but do you really?

Reply:
This app allows you to launch the appimages that the user "integrates" into the system menu.
The interface is designed to look like an app store (sort of), with big buttons to open or uninstall apps.

Before making a file executable, the app shows the md5 hash of the file and the user needs to manually check a label to mark the app as trusted.

This is all this permission it is used for.

Reply:
@hfiguiere I applied for an exception. 
Do you even accept these requests if I provide evidence I need this permission?